osu.Game/Screens/Select/LocalScoreDeleteDialog.cs

```diff
@@ -0,0 +1,62 @@
+// Copyright (c) ppy Pty Ltd <contact@ppy.sh>. Licensed under the MIT Licence.
+// See the LICENCE file in the repository root for full licence text.
+
+using osu.Framework.Allocation;
+using osu.Game.Overlays.Dialog;
+using osu.Game.Scoring;
+using System;
+using System.Threading.Tasks;
+using osu.Framework.Graphics.Sprites;
+using osu.Framework.Logging;
+
+namespace osu.Game.Screens.Select
+{
+    public class LocalScoreDeleteDialog : PopupDialog
+    {
+        private ScoreManager scoreManager;
+
+        public LocalScoreDeleteDialog(ScoreInfo score)
+        {
+            try
+            {
+                string accuracy = string.Format(score?.Accuracy % 1 == 0 ? @"{0:P0}" : @"{0:P2}", score?.Accuracy);
+
+                BodyText = $@"{score} {Environment.NewLine} Rank: {score.Rank} - Max Combo: {score.MaxCombo} - {accuracy}";
```

Comment:
I'd make this match beatmap deletion, rather than listing rank/combo/accuracy/etc. Here's a comparison:

![image](https://user-images.githubusercontent.com/1329837/71144093-26b7b580-2260-11ea-9264-6d2d9ae8ed1c.png)

![image](https://user-images.githubusercontent.com/1329837/71144107-33d4a480-2260-11ea-9291-3016e5ba5b88.png)

Reply:
I would argue that the score information are necessary since this feature is deleting individual local score. Please let me know if there are any specific reasons that you would recommend your suggestion.

Reply:
at the very least, i would consider having the score information display in a more digestable format

"smoogipoo's 68.25% D Rank on TRF - Survival dAnce ~no no cry more~ [Insane]" might be an immediate improvement to the current format

Reply:
Yeah, I agree. I will make the changes accordingly.